### DISCUSSION TITLE
Any plans to add an auto-resizing textarea?

### DISCUSSION THREAD

**Discussion by pikhosh** [skeletonlabs/skeleton#53] [Q&A] [answered]
Since it has been mentioned that you don't intend to implement the inputs and you suggest that we use the Tailwind forms plugin, I wanted to ask if you have a plan to implement an auto-resizing `textarea`? Because I didn't see anything like this in the plugin or in Skeleton.

**Comment by endigo9740** [comment, marked as answer]
Hey Pikhosh, do you have an example of this? When I hear resizable textarea I think of Chrome, which has had that by built-in by default for years. Perhaps this is something that is not available cross-browser or cross-OS?

Just so you're aware, me and Thomas (Skeleton's other core contributor) are aiming to meet and to flush out a roadmap for the project starting early next week. This will help solidify the goals for the next few months. Of course we'll be taking into account feedback from you and others in the community.

I'm hesitant to provide specifics yet, as I don't want to over promise. However, I do have some exciting ideas regarding forms and inputs that you may find useful.

When the roadmap is ready you can expect to see it on the Wiki section here on Github. Likewise I'll generate a new Discussion to gather feedback. I highly encourage you to review that and let us know your thoughts! We're trying to build this for the community after all!

**Reply by pikhosh** [reply]
> Hey Pikhosh, do you have an example of this? When I hear resizable textarea I think of Chrome, which has had that by built-in by default for years. Perhaps this is something that is not available cross-browser or cross-OS?

Something like this: [NaiveUI](https://codesandbox.io/s/8lwg71)

**Reply by endigo9740** [reply]
Hmm that's interesting, I'll add this to my notes to review next week. Thanks!

**Reply by thomasbjespersen** [reply]
That does look quite interesting. Definitely worth to review.

**Reply by pikhosh** [reply]
> Hmm that's interesting, I'll add this to my notes to review next week. Thanks!

It was actually easier than I thought. 😅
If it's ok, I can add this, but I don't know if it's worth it when there are no other input components...
